### PR TITLE
Stat tracker

### PR DESCRIPTION
**Pull request by Henchworm**
Stat tracker git fix

### CODE REVIEW

**Review by jacksonvaldez — COMMENTED** [review]
Looks good!

File: spec/fixtures/spec teams.csv [hunk 1]
```diff
@@ -0,0 +1,33 @@
+team_id,franchiseId,teamName,abbreviation,Stadium,link
```
Comment (jaw772) [inline spec/fixtures/spec teams.csv:1]: good use of fixtures

**Review by jacksonvaldez — APPROVED** [review]

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+require 'simplecov'
```
Comment (jaw772) [inline spec/stat_tracker_spec.rb:1]: simpleCov looks good

**Review by jaw772 — APPROVED** [review]

**Review by jacksonvaldez — APPROVED** [review]

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  def initialize(games, teams, game_teams)
+    @games      = games#games-hash value
+    @teams      = teams #same as above
+    @game_teams = game_teams #same as above
+  end
+
+
+  def self.from_csv(locations)
+    games = CSV.read(locations[:games], headers: true)
```
Comment (tjroeder) [inline lib/stat_tracker.rb:15]: Nice application from the CSV lesson! Looks good before we add the data to the classes.

**Review by tjroeder — APPROVED** [review]
Looks good for the CSV importation.

**Pull request merged**